src/jupyblog/models.py

```diff
@@ -126,10 +126,14 @@ class Settings(BaseModel):
 
     execute_code : bool, default=True
         Execute code snippets.
+
+    version : str, default=''
+        Jupyblog version used to create the rendered file
     """
     serialize_images: bool = False
     allow_expand: bool = False
     execute_code: bool = True
+    version: str = ''
```

Comment:
let's put this under a `metadata` section, you'll have to create a new Metadata model and put version there, check pydantic's docs for details